.changeset/witty-jokes-hammer.md

```diff
@@ -0,0 +1,10 @@
+---
+'@axis-backstage/plugin-jira-dashboard-backend': minor
```

Comment:
As the PR is currently this is a breaking change, since it changes both the response and arguments for the "searchJira" call.

Reply:
Resolved :)